src/api/todos.ts

```diff
@@ -0,0 +1,22 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const createTodos = (userId: number, todo: Todo) => {
+  return client.post<Todo>(`/todos?userId=${userId}`, todo);
+};
+
+export const updateTodosTitle = (todoId: number, title: string) => {
+  return client.patch<Todo>(`/todos/${todoId}`, { title });
+};
+
+export const updateTodosCompleted = (todoId: number, completed: boolean) => {
+  return client.patch<Todo>(`/todos/${todoId}`, { completed });
+};
```

Comment:
could we combine it together to one common method `updateTodo` and pass necessary object?